ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/utils/getImageScanMessage.test.ts

```diff
@@ -0,0 +1,76 @@
+import { imageScanMessages } from 'messages/vulnMgmt.messages';
```

Comment:
I copied over this function from `src/Containers/VulnMgmt/VulnMgmt.utils/getImageScanMessage.test.ts` instead of moving and reusing the function. When we migrate over to solely using VM 2.0 and removing the old VM 1.0, we can just delete the contents of `VulnMgmt`, which will delete the old file anyways.